doc-content/enterprise-only/installation/controller-jws-install-proc.adoc

```diff
@@ -136,36 +137,42 @@ endif::PAM[]
 ifdef::DM[]
 [source]
 ----
-set CATALINA_OPTS= -Xmx1024m 
- -Dorg.jbpm.server.ext.disabled=true 
- -Dorg.jbpm.ui.server.ext.disabled=true 
- -Dorg.jbpm.case.server.ext.disabled=true 
- -Dorg.kie.server.controller.user=<CONTROLLER_USER> 
- -Dorg.kie.server.controller.pwd=<CONTROLLER_PWD> 
+set CATALINA_OPTS= -Xmx1024m
+ -Dorg.jbpm.server.ext.disabled=true
+ -Dorg.jbpm.ui.server.ext.disabled=true
+ -Dorg.jbpm.case.server.ext.disabled=true
+ -Dorg.kie.server.controller.user=<CONTROLLER_USER>
+ -Dorg.kie.server.controller.pwd=<CONTROLLER_PWD>
  -Dorg.kie.server.id=<KIE_SERVER_ID>
  -Dorg.kie.server.location=http://<HOST>:<PORT>/kie-server/services/rest/server
- -Dorg.kie.server.controller=http://<HOST>:<PORT>/controller/rest/controller
+ -Dorg.kie.server.controller=http://<CONTROLLER_HOST>:<CONTROLLER_PORT>/controller/rest/controller
 ----
 endif::DM[]
++
+In the preceding examples, replace the following variables:
```

Comment:
Is there another example after this that is part of a sequence? If not should we just say "previous" instead of "preceding"?

Reply:
Well, I confess I have a personal bias for that form for no good reason then consistency and habit. I asked Stetson about this and here is his take:
"Preceding is "correct" when it immediately follows. Previous is "correct" if it's farther away. So in the example you're giving, if I had to choose between the two, I would go with preceding. But I just don't use it at all anyway in such cases. Not an issue of correctness as much as superfluity."

So I guess I was correct but I take his point about superfluity.